CTY-4_GHGemissions.ipynb

```diff
@@ -1,35 +1,22 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #30.                pop_img = ghspop_img.divide(ghspop_img.pixelArea())
</pre><p>Is this being used?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wri/SCL-cities/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>